modules/020-deckhouse/hooks/internal/updater/notification.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2022 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package updater
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+
+	"github.com/deckhouse/deckhouse/modules/020-deckhouse/hooks/internal/apis/v1alpha1"
+)
+
+type NotificationConfig struct {
+	WebhookURL              string
+	MinimalNotificationTime v1alpha1.Duration
+}
+
+func ParseNotificationConfigFromValues(input *go_hook.HookInput) *NotificationConfig {
+	webhook, ok := input.Values.GetOk("deckhouse.update.notification.webhook")
+	if !ok {
+		return nil
+	}
+
+	var minimalTime v1alpha1.Duration
+	t, ok := input.Values.GetOk("deckhouse.update.notification.minimalNotificationTime")
+	if ok {
+		err := json.Unmarshal([]byte(t.Raw), &minimalTime)
+		if err != nil {
+			panic(err)
+		}
+	}
+
+	return &NotificationConfig{
+		WebhookURL:              webhook.String(),
+		MinimalNotificationTime: minimalTime,
+	}
+}
+
+func sendWebhookNotification(webhookURL string, data webhookData) error {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
In another products, I always see that this `InsecureSkipVerify` are passing with URL as separate parameter. Client could choice skipping verify. Or we should say about skip TLS check in [documentation](https://github.com/deckhouse/deckhouse/pull/2131/files#diff-6062960603271f16b22204e3e46807bc9333f9ad4d68adc866826e17064be2e5R112), i think.

Reply:
Separated with config